origin-bridge/src/logger.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const Logger = require('logplease')
+Logger.setLogLevel('DEBUG')
+module.exports = Logger.create('origin-bridge', {
+  color: Logger.Colors.Yellow
```

Comment:
nit: I also tend to disable timestamp logging by setting option `showTimestamp: false` since we already get timestamp from GCP logs in prod.

Reply:
Good plan!